### PR TITLE
Add stats calculation for requests under task

### PR DESCRIPTION
**Pull request by JingJZ160**
Stats calculation for requests given a task id.

### CODE REVIEW

File: ar/data.py [hunk 1]
```diff
@@ -663,3 +640,40 @@ def export_labeled_examples(task_id, outfile=None):
...
+        return "<Context Id {}, Annotation Value {}>".format(self.context_id,
+                                                             self.value)
+
+    def __eq__(self, other):
+        if isinstance(other, ContextAndAnnotationValuePair):
+            return self.context_id == other.context_id and self.value ==  \
+                   other.value
+        return NotImplementedError("{} is not an instance of {}".format(
+            str(other), self.__class__.__name__
+        ))
```
Comment (edu-gp) [inline ar/data.py:679]: I haven't looked deep into where `UserNameAndIdPair` and `ContextAndAnnotationValuePair` are used. But I was wondering, would a `namedtuple` suffice?

e.g.
```
UserNameAndIdPair = namedtuple("UserNameAndIdPair", ['id','username'])
blah = UserNameAndIdPair(12, 'foobar')
hash(blah)
```

File: tests/unit/test_data.py [hunk 2]
```diff
@@ -210,3 +211,93 @@ def populate_annotations():
             tuple(sorted([username2, username3])): res3
         }
     }
+
+
+def test_compute_annotation_request_statistics(dbsession):
+    username1 = "ooo"
+    username2 = "ppp"
+    text1 = "whatever"
+    text2 = "SQL is no fun."
```
Comment (edu-gp) [inline tests/unit/test_data.py:220]: lol what

File: ar/data.py (hunk 1 above)
Comment (JingJZ160) [reply ar/data.py:679]: Oh nice. I'll use `namedtuple` instead :)

File: tests/unit/test_data.py (hunk 2 above)
Comment (JingJZ160) [reply tests/unit/test_data.py:220]: 😜

**Pull request merged**